happypose_ros/happypose_ros/happypose_node.py

```diff
@@ -188,6 +191,12 @@ def __init__(self, node_name: str = "happypose_node", **kwargs) -> None:
             qos_profile=qos_profile_system_default,
             qos_overriding_options=QoSOverridingOptions.with_default_policies(),
         )
+        self._seg_masks_publisher = self.create_publisher(
```

Comment:
Maybe there should be a parameter enabling this feature?

Reply:
Indeed, will do!

Reply:
done